.gitignore

```diff
@@ -19,6 +19,7 @@
 package-lock.json
 
 # IDE - VSCode
+.vscode
```

Comment:
leftover?

Reply:
Actually it didn't ignore vscode folder that's why I put it.